src/components/NormalDom.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChakraProvider } from "@chakra-ui/react";
+
+import React from "react";
+import ReactDOM from "react-dom";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+export function NormalDom({
+  parentElement,
+  position = "beforebegin",
+  children,
+}: {
+  parentElement: Element;
+  position?: InsertPosition;
+  children: React.ReactNode;
+}) {
+  const [myModal] = React.useState(() => document.createElement("my-modal"));
```

Comment:
Mozna bych sem netahal "myModal" ale zkusil bych neco neutralniho (ac chapu, ze to pouzivame prave pro ten modal).